tests/frameworks/OSXShims/include/IwMalloc.h

```diff
@@ -0,0 +1,67 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
```

Comment:
how did you even find this I haven't even sent out the mail yet :(

---
In reply to: [70345994](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70345994) [](ancestors = 70345994)